packages/protoc-gen-connect-es/README.md

```diff
@@ -0,0 +1,204 @@
+# @bufbuild/protoc-gen-connect-es
+
+The code generator for connect-es, a simple library to work with servers and clients
```

Comment:
```suggestion
The code generator for Connect, a simple library to work with servers and clients
```

This is shown on npmjs.com, so calling it just "Connect" doesn't seem totally out of place. Gives us a bit more wiggle room for the transition. WDYT?

Reply:
sgtm.  updated.